_posts/2021-02-02-sc20-report.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: RSE-HPC-2020 Workshop Report
+tags: [conference]
+posted_by: Charles Ferenbaugh, Daniel S. Katz, Sandra Gesing, and Simon Hettrick
+---
+
+We were excited to hold the inaugural [Research Software Engineers in HPC
+Workshop (RSE-HPC-2020)](https://us-rse.org/rse-hpc-2020/) as part of the
+[SC20](https://sc20.supercomputing.org/) conference this year! The half-day,
+virtual event featured 18 speakers and over 100 participants.
+
+The opening session, chaired by Charles Ferenbaugh, began with keynote addresses
+from Simon Hettrick in the UK and Frank Löffler in Germany. They spoke on the
+growth of the RSE movements in their respective countries. Though the name
+"Research Software Engineer" is only about eight years old, the role is much
+older. In those countries (and many others), when RSE events were first
+organized, many RSEs joined in right away and communities grew quickly. Many who
+joined said they were excited to have "found their tribe."
```

Comment:
Maybe "found their people" ? I listened to a podcast this morning about a woman that started a running community called "Tribe" and it made several indigenous communities upset and she changed it. So we should avoid that :) I actually have heard and said "found my people" many times, but I've never heard someone say tribe.

Reply:
Episode -> https://runningmagazine.ca/shakeout-podcast/heather-gardner-on-cultivating-an-inclusive-run-crew-community-and-growing-through-discomfort/

Reply:
I hear "found my tribe" fairly often, and that is verbatim what was said at the workshop, but you've raised a good point.  I'm inclined to say that, since it is an actual quote and it's a relatively minor part of the article, we leave it as is.  That is a different case from using it as the name of a group.  Does that sounds sensible?

Reply:
I think we should be aware that the language we use can be offensive, even ifothers have already used it, and nobody has been offended (yet). The woman in the podcast was using tribe only to mean community, and there were indigenous folks that were upset. If there is any chance of making someone upset reading the post because of this term, given that there is an easy replacement that has also been used (found their people) I think it's better to err on the conservative side and use it. That's just how I feel about it.

Reply:
Before listening to this podcast I wouldn't have been aware that people can be offended by using that term to mean community, but now I am.

Reply:
I suppose this makes sense.  I'll change it.